tools/lint/cmd_data.c

```diff
@@ -675,11 +739,53 @@ process_data(struct ly_ctx *ctx, enum lyd_type type, uint8_t merge, LYD_FORMAT o
 int
 cmd_data_process(struct ly_ctx *ctx, struct yl_opt *yo)
 {
-    /* parse, validate and print data */
-    if (process_data(ctx, yo->data_type, yo->data_merge, yo->data_out_format, yo->out, yo->data_parse_options,
-            yo->data_validate_options, yo->data_print_options, &yo->data_operational, &yo->reply_rpc,
-            &yo->data_inputs, &yo->data_xpath)) {
-        return 1;
+    if (yo->data_ext) {
+        struct lyd_node *tree = NULL;
+        struct cmdline_file *input_f = NULL;
+        struct lys_module *module = NULL;
+        uint32_t idx = 0;
+
+        while ((module = ly_ctx_get_module_iter(ctx, &idx))) {
+            if (!strcmp(module->prefix, yo->prefix)) {
```

Comment:
It is not safe to assume the prefix is unique, only the module name must be unique so use that instead.